cogeo_mosaic/backends/base.py

```diff
@@ -146,4 +137,6 @@ def _update(
         )
         self._update_metadata(bounds, version)
 
+        if self.path:
+            self.write()
```

Comment:
This is mostly to support `virtual` mosaic when doing 
```python
with MosaicBackend(None, mosaic_def=mymosaic) as mosaic:
     mosaic.update([1.tif])
```

Reply:
Does every backend have a `path` attribute?

Reply:
yes starting with this PR